installer.sh

```diff
@@ -47,14 +47,14 @@ URL_SERVICE_HASSIO="${URL_RAW_BASE}/hassio-supervisor.service"
 URL_APPARMOR_PROFILE="https://version.home-assistant.io/apparmor.txt"
 
 # Check env
-command -v systemctl > /dev/null 2>&1 || MISSING_PACKAGES+=("systemd")
-command -v nmcli > /dev/null 2>&1 || MISSING_PACKAGES+=("NetworkManager")
-command -v apparmor_parser > /dev/null 2>&1 || MISSING_PACKAGES+=("AppArmor")
-command -v docker > /dev/null 2>&1 || MISSING_PACKAGES+=("docker")
-command -v jq > /dev/null 2>&1 || MISSING_PACKAGES+=("jq")
-command -v curl > /dev/null 2>&1 || MISSING_PACKAGES+=("curl")
-command -v avahi-daemon > /dev/null 2>&1 || MISSING_PACKAGES+=("avahi")
-command -v dbus-daemon > /dev/null 2>&1 || MISSING_PACKAGES+=("dbus")
+command -v systemctl > /dev/null 2>&1 || MISSING_PACAKGES+=("systemd")
```

Comment:
Your `MISSING_PACAKGES` should be  `MISSING_PACKAGES`

Reply:
> Your `MISSING_PACAKGES` should be `MISSING_PACKAGES`

Haha thanks! Good eyes! 👁️👁️ I guess I had made my feature branch before the last PR to fix the typos was approved and merged. Fixing it up now!